src/runtime/utils/entry.mjs

```diff
@@ -56,3 +56,57 @@ export function initReducedView() {
     tmp.remove();
   });
 }
+
+export async function hasBatteryPerformanceIssue(videoBlob) {
+  await isBatteryLow();
+  await canVideoPlay(videoBlob);
+}
+
+/**
+ * Checks if battery still has enough energy.
+ * This check is for Chrome and all other browsers that support this setting.
+ *
+ * Condition is: The device is not charging and Battery is below <= 20%.
+ * @see https://blog.google/products/chrome/new-chrome-features-to-save-battery-and-make-browsing-smoother/
+ * @see https://developer.chrome.com/blog/memory-and-energy-saver-mode/
+ **/
+async function isBatteryLow() {
+  const MIN_BATTERY_LEVEL = 0.2;
+
+  try {
+    const battery = await window.navigator.getBattery();
+    if (!battery.charging && battery.level <= MIN_BATTERY_LEVEL) {
+      throw new Error('Battery is low.');
+    }
+  } catch (error) {
+    // Ignore Check
```

Comment:
silent catch?

Reply:
The unnecessary catch has been removed.

In the entry now `console.warn` are built in, these show the respective case as a warning.

![image](https://github.com/GrabarzUndPartner/nuxt-speedkit/assets/8287751/062e250b-782f-4b25-9f1b-80889a903612)